front/src/components/boxs/moon/style.css

```
@@ -0,0 +1,32 @@
+.moonPhaseName {
+  font-weight: 600;
+}
+
+.moonDetails {
+  flex: 1 1 240px;
+  min-width: 0;
+}
+
+.moonRowValue {
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)